crates/store/re_remote_store_types/src/lib.rs

```diff
@@ -0,0 +1,339 @@
+//! This crate contains generated types for the remote store gRPC service API.
+//! Generation is done using the `re_remote_store_types_builder` crate.
+//!
+//! We want clear separation between 'internal' types and gRPC types and don't want
+//! to use gRPC types in the rerun viewer codebase. That's why we implement all the
+//! necessary conversion code (in the form of `From` and `TryFrom` traits) in this crate.
+//!
+
+// Ignoring all warnings for the auto-generated code.
+#![allow(clippy::doc_markdown)]
+#![allow(clippy::derive_partial_eq_without_eq)]
+#![allow(clippy::enum_variant_names)]
+#![allow(clippy::unwrap_used)]
+#![allow(clippy::wildcard_imports)]
+#![allow(clippy::manual_is_variant_and)]
+/// Generated types for the remote store gRPC service API v0.
+pub mod v0 {
+    #[path = "../v0/rerun.remote_store.v0.rs"]
+    mod _v0;
+
+    pub use self::_v0::*;
+
+    // ==== below are all necessary transforms from internal rerun types to protobuf types =====
+
+    use std::collections::BTreeSet;
+
+    #[derive(Debug, thiserror::Error)]
+    pub enum TypeConversionError {
+        #[error("missing required field: {0}")]
+        MissingField(&'static str),
+    }
+
+    impl From<re_log_types::ResolvedTimeRange> for TimeRange {
+        fn from(time_range: re_log_types::ResolvedTimeRange) -> Self {
+            Self {
+                start: time_range.min().as_i64(),
+                end: time_range.max().as_i64(),
+            }
+        }
+    }
+
+    impl TryFrom<Query> for re_dataframe::external::re_chunk_store::QueryExpression {
```

Comment:
Here and everywhere you shouldn't need `re_dataframe::external::re_chunk_store::QueryExpression`, simply `re_dataframe::QueryExpression`.

If that doesn't work then we have a re-export problem, and it's important to be fixed as these are supposed to be sane-to-use public APIs.

Reply:
missing exports addressed via https://github.com/rerun-io/rerun/pull/7709